composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
+      "post-autoload-dump": [
+        "php -r \"symlink('../../../../../../vendor/swagger-api/swagger-ui', './src/Bridge/Symfony/Bundle/Resources/public/swagger-ui');\""
```

Comment:
I think you meant to symlink the `dist` directory?

https://github.com/swagger-api/swagger-ui/tree/master/dist